### PR TITLE
Club users and test

### PR DESCRIPTION
**Pull request by mjtorn**
Can we leave integration-type tests for later? That is having two clubs, one user (membership with privileges) and making sure he can't alter the other club.

Otherwise anything ok here?

### CODE REVIEW

File: barbados/barbadosdb/models.py [hunk 1]
```diff
@@ -168,6 +119,18 @@ def __str__(self):
         return self.name
 
 
+class Membership(models.Model):
+    """Membership (not the sailable) in club
+    """
+
+    user = models.ForeignKey(User)
+    club = models.ForeignKey('Club')
```
Comment (greygriffin) [inline barbados/barbadosdb/models.py]: Put class Membership after class Club, and you won't need quotes around 'Club'.

File: barbados/barbadosdb/models.py [hunk 2]
```diff
@@ -168,6 +119,18 @@ def __str__(self):
         return self.name
 
 
+class Membership(models.Model):
+    """Membership (not the sailable) in club
+    """
+
+    user = models.ForeignKey(User)
```
Comment (greygriffin) [inline barbados/barbadosdb/models.py]: Should we use related_name='users' as in hifistely-PR? And likewise for club. But then you couldn't ask for a club's name without getting a list of urls for each of its members.

**Comment by mjtorn** [comment]
I think what you're going for is naming that in Club? The user attribute there could take `related_name='clubs'` so we could access `u.clubs` instead of `u.club_set`.

I don't like it, because attribute-managers named `_set` are everywhere

**Comment by greygriffin** [comment]
Looks good to me

**Pull request merged**